### PR TITLE
Add `org_name` config param

### LINKED ISSUE #4: Allow using `org_name` instead of `ins_id`
When calling `NewClient`, the exact installation ID can be calculated from `org_name`.

### PR DESCRIPTION
**Pull request by lego963**
Add `org_name` config param as alternative of `ins_id`

Resolves: #4

### CODE REVIEW

File: github/integration_test.go [hunk 1]
```diff
@@ -1,3 +1,4 @@
+//go:build integration
```
Comment (outcatcher) [inline github/integration_test.go]: This seems to be a redundant change

File: github/backend_test.go [hunk 2]
```diff
@@ -10,7 +10,7 @@ import (
 	"github.com/hashicorp/vault/sdk/logical"
 	"gotest.tools/assert"
 
-	hclog "github.com/hashicorp/go-hclog"
+	"github.com/hashicorp/go-hclog"
```
Comment (outcatcher) [inline github/backend_test.go:13]: This seems to be a redundant change

File: github/client.go [hunk 3]
```diff
@@ -177,6 +187,75 @@ func (c *Client) Token(ctx context.Context, opts *tokenOptions) (*logical.Respon
...
+
+	var instResult []Installation
+	if err := json.NewDecoder(res.Body).Decode(&instResult); err != nil {
+		return 0, err
+	}
+
+	var instID int
+	for _, v := range instResult {
+		if v.Account.Login == config.OrgName {
+			instID = v.ID
```
Comment (outcatcher) [inline github/client.go]: You can return from here and make replace code after the cycle with just
```go
return 0, fmt.Errorf("installation ID for the organization wasn't found")
```

Maybe also reword error message? Mentioning app not installed in the organization.

File: github/path_config_test.go [hunk 4]
```diff
@@ -147,6 +147,41 @@ func testBackendPathConfigCreateUpdate(t *testing.T, op logical.Operation) {
 		assert.DeepEqual(t, testBaseURLValid, config.BaseURL)
 	})
 
+	t.Run("Exist Organization", func(t *testing.T) {
```
Comment (outcatcher) [inline github/path_config_test.go]: You can use sub-tests for those two `Exist` tests, it seems only configuration differs, but not the logic.

File: github/path_config_test.go [hunk 5]
```diff
@@ -181,6 +216,26 @@ func testBackendPathConfigCreateUpdate(t *testing.T, op logical.Operation) {
 		assert.Assert(t, is.Nil(resp))
 	})
 
+	t.Run("FailedStoragePersist Organization", func(t *testing.T) {
```
Comment (outcatcher) [inline github/path_config_test.go]: You can use sub-tests for those two `FailedStoragePersist` tests, it seems only configuration differs, but not the logic.

**Review by outcatcher — CHANGES_REQUESTED** [review]

File: github/backend_test.go (hunk 2 above)
Comment (lego963) [reply github/backend_test.go:13]: this is redundant alias

**Review by outcatcher — APPROVED** [review]

**Pull request merged**